builder/packages_image.go

```diff
@@ -260,17 +263,31 @@ func (p *PackagesImageBuilder) generateDockerfile(baseImage string, packages mod
 
 // GetRolePackageImageName generates a docker image name for the amalgamation for a role image
```

Comment:
Fixed in 77525b733063a37a060b509a26f72394c892eaa0 (`GetPackagesLayerImageName()`)